.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "Feature request \U0001F4A1"
+about: Suggest an idea for this project
+labels: feature
+
+---
+
+### Motivation
+<!-- A clear and concise description of what the motivation for the new feature is, and what problem it is solving. -->
+
+
+### User Story
+<!-- A clear description of the User Stories that should be achieved by the new feature. The User Stories should follow this pattern: 
+As a [type of user] I want [goals or objectives] so that [values or benefits]. -->
+
+
+### Proposed Solution
+<!-- A clear and concise description of the feature you would like to add, and how it solves the motivating problem. -->
+
+
+### Alternatives
+<!-- A clear and concise description of any alternative solutions or features you've considered, and why you're proposed solution is better. -->
+
+
+### Additional Context
+<!-- Add any other information or screenshots about the feature request here. -->
+
+
+### Design Requirements
+<!-- If the customization includes input from our design team, the detailed requirements will be collected here. Note: These will exist mainly in German to simplify internal communication. -->
```

Comment:
Are we going to be working with the design team on this? (Genuine question, I do not know)

Reply:
I think we'll be using the existing mockups that we already have from the pre-existing Figma file. I was planning to add screenshots for every issue that has something to do with templates, but if everyone has access to the link, I guess you're right and we can skip this part

Reply:
Hmm, the "Additional Context" section already mentiones screenshots. I don't think we need a separate section for this..

Reply:
That is a valid point. However I like that the new section is more prominent (and hopefully reminds people to also upload a screenshot more often) than just Additional Context. Can I convince you to remove the "or screenshots" from Additional Context and keep it as two sections. Additional Context then being more for written, informational background context and Screenshots solely for screenshots. If not, I'm also fine with removing the section

Reply:
Sure, would be fine for me.